PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/selection/SelectionManager.java

```diff
@@ -123,4 +130,30 @@ private void clearDescription() {
     });
     timeline.play();
   }
+
+  /**
+   * Set the IPhylogeneticTreeNode which is shown in the top graph.
+   *
+   * @param topNode the IPhylogeneticTreeNode which is shown in the top graph.
+   */
+  public void setTopGraphNode(IPhylogeneticTreeNode topNode) {
+    topGraphNode.set(topNode);
+  }
+
+  /**
+   * Set the IPhylogeneticTreeNode which is shown in the bottom graph.
+   *
+   * @param bottomNode the IPhylogeneticTreeNode which is shown in the bottom graph.
+   */
+  public void setBottomGraphNode(IPhylogeneticTreeNode bottomNode) {
+    bottomGraphNode.set(bottomNode);
+  }
+
```

Comment:
If you write javadoc for the setters, it would be nice if you also do the getters